rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -949,7 +949,7 @@ SecRule REQUEST_HEADERS:Content-Type "@rx ^[^;\s]+" \
 #
 # Restrict charset parameter within the content-type header
 #
-SecRule REQUEST_HEADERS:Content-Type "@rx charset\s*=\s*([^;\s]+)" \
+SecRule REQUEST_HEADERS:Content-Type "@rx charset\s*=\s*[\"\']?([^;\"\'\s]+)" \
```

Comment:
Can you remove the escaping from `'`?

Reply:
Removed escape chars from single quotes.